actor.go

```diff
@@ -69,3 +69,14 @@ func (actor *Actor) IsLeader() bool {
 	}
 	return false
 }
+
+// Leader returns the LibP2P ID of the Raft leader.
+func (actor *Actor) Leader() peer.ID {
+	// Leader as returned by Libp2pTransport.LocalAddr()
+	raftLeader := actor.Raft.Leader()
+	peerID, err := peer.IDB58Decode(raftLeader)
+	if err != nil {
+		panic(err)
```

Comment:
Sketchy to panic, but alright. Is there no possible situation we could end up with the leader reporting an invalid peer ID as their name?